main.rb

```diff
@@ -0,0 +1,158 @@
+require './app'
+
+class Main
+  def initialize
+    @app = App.new
+    @option = ''
+    @type = ''
+    @age = ''
+    @name = ''
+    @permission = ''
+    @specialization = ''
+    @title = ''
+    @author = ''
+    @book = ''
+    @person = ''
+    @date = ''
+    @id = ''
+  end
+
+  def menu_content
+    puts
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
```

Comment:
- [ ] Please kindly make sure your menu is correct when I select option `5. create a rental` it shows the list of books and people, it's supposed to create a rental not to list all books and people 🙃 😉
<img width="924" alt="Screen Shot 2022-09-01 at 13 42 03" src="https://user-images.githubusercontent.com/74501523/187909469-e024c2a8-421c-47aa-aa11-f42564334700.png">